app/src/main/kotlin/me/myshows/android/NetModule.kt

```diff
@@ -0,0 +1,55 @@
+package me.myshows.android
+
+import com.fasterxml.jackson.databind.BeanDescription
+import com.fasterxml.jackson.databind.DeserializationConfig
+import com.fasterxml.jackson.databind.JsonDeserializer
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.deser.BeanDeserializerModifier
+import com.fasterxml.jackson.databind.module.SimpleModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import dagger.Module
+import dagger.Provides
+import me.myshows.android.model.EpisodeRating
+import me.myshows.android.model.serialization.EpisodeRatingDeserializer
+import okhttp3.JavaNetCookieJar
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import java.net.CookieManager
+import javax.inject.Singleton
+
+@Module
+open class NetModule {
+
+    @Singleton
+    @Provides
+    open fun objectMapper(): ObjectMapper {
```

Comment:
Why `ObjectMapper` in `NetModule`? I see that we use it only for API client, however, it seems a bit strange that `NetModule` provides such kind of object.